tanis2000/rust-sdl2-wasm
# Update deps & build

Tested locally on mac os 13.2.1 with SDL2 and emsdk installed from homebrew.

* update to work with latest versions of rustc and sdl2
* use gl crate instead of running gl-generator locally
* use dynamically linked sdl2 instead of building locally
* remove emcc clamp flag that is no longer needed and breaks build
